pkg/vpcmodel/responsive_test.go

```diff
@@ -0,0 +1,133 @@
+/*
+Copyright 2023- IBM Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+package vpcmodel
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/np-guard/models/pkg/connection"
+	"github.com/np-guard/models/pkg/netp"
+)
+
+func newTCPConn(t *testing.T, srcMinP, srcMaxP, dstMinP, dstMaxP int64) *connection.Set {
```

Comment:
isn't this a duplication of `newTCPSet` from `responsive.go`?

Reply:
not exactly. `newTCPSet`  generates a `TCP` connection with all ports.
I moved `newTCPConn` and have `newTCPSet` use it.